configure.ac

```diff
@@ -1154,6 +1167,19 @@ TS_CHECK_ZLIB
 TS_CHECK_LZMA
 
 #
+# System LuaJIT
+#
+
+if test "x$with_system_luajit" == "xyes"; then
+    saveLIBS=$LIBS
+    LIBS=""
+    AC_CHECK_LIB(luajit-5.1, luaL_newstate, [],
```

Comment:
We can be sure the luajit library, even though it's in version "2.x", will remain being libluajit-5.1 ? And this is consistent across platforms that supports it? I did check my F25 installation, and it does exist there.